earthfile2llb/interpreter.go

```diff
@@ -171,19 +171,23 @@ func (i *Interpreter) handleStatement(ctx context.Context, stmt spec.Statement)
 	ctx = ContextWithSourceLocation(ctx, stmt.SourceLocation)
 	if stmt.Command != nil {
 		return i.handleCommand(ctx, *stmt.Command)
-	} else if stmt.With != nil {
+	}
+	if stmt.With != nil {
 		return i.handleWith(ctx, *stmt.With)
-	} else if stmt.If != nil {
+	}
+	if stmt.If != nil {
 		return i.handleIf(ctx, *stmt.If)
-	} else if stmt.For != nil {
+	}
+	if stmt.For != nil {
 		return i.handleFor(ctx, *stmt.For)
-	} else if stmt.Wait != nil {
+	}
+	if stmt.Wait != nil {
 		return i.handleWait(ctx, *stmt.Wait)
-	} else if stmt.Try != nil {
+	}
+	if stmt.Try != nil {
 		return i.handleTry(ctx, *stmt.Try)
-	} else {
-		return i.errorf(stmt.SourceLocation, "unexpected statement type")
 	}
+	return i.errorf(stmt.SourceLocation, "unexpected statement type")
```

Comment:
I believe that the intention here is that only one of these is set at a time - hence the `else`s. Is this change intentional?

Reply:
Oh yeah - they end in return. That's fine then.